README.md

```diff
@@ -153,19 +158,25 @@ single Chain Table value to 8. The prefix is used to write the actual length of
 the data stored inside a block.
 
 Therefore:
+
 - given $N$ the number of keywords used, the size of the Entry Table is given
   by (in bytes):
+
 ```math
 L_{entry~table} = (L_{uid} + C_e + L_{K_{w_i}} + L_{H_{w_i}} + L_{uid}) \cdot N
-       		= 140 \cdot N
+         = 140 \cdot N
```

Comment:
Why is not aligned anymore?

Reply:
because tabs were used here. Inserting new spaces to align the =